ui/tests/pages/jobs/list.js

```diff
@@ -66,10 +64,9 @@ export default create({
   pageSizeSelect: pageSizeSelect(),
 
   facets: {
-    namespace: singleFacet('[data-test-namespace-facet]'),
-    type: multiFacet('[data-test-type-facet]'),
-    status: multiFacet('[data-test-status-facet]'),
-    datacenter: multiFacet('[data-test-datacenter-facet]'),
-    prefix: multiFacet('[data-test-prefix-facet]'),
+    namespace: hdsFacet('[data-test-facet="Namespace"]'),
+    type: hdsFacet('[data-test-facet="Type"]'),
+    status: hdsFacet('[data-test-facet="Status"]'),
+    nodePool: hdsFacet('[data-test-facet="NodePool"]'),
```

Comment:
We don't have datacenter or prefix dropdowns anymore, nor do we have priority or task group columns in the table itself.